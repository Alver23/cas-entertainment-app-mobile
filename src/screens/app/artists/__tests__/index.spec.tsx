// Dependencies
import React from 'react';
import { shallow, ShallowRendererProps } from 'enzyme';

// Screen
import { ArtistScreen } from '../index';

describe('<ArtistScreen />', () => {
  let component: ShallowRendererProps;

  beforeEach(() => {
    component = shallow(<ArtistScreen />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
