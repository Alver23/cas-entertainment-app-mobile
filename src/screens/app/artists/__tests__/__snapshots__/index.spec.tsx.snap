// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArtistScreen /> should save a snapshot of the component 1`] = `
<LayoutView>
  <View
    style={
      Object {
        "marginHorizontal": 24,
        "marginVertical": 12,
      }
    }
  >
    <Themed.Text
      style={
        Object {
          "fontSize": 22,
        }
      }
    >
      Artistas
    </Themed.Text>
  </View>
  <ArtistContainer />
  <ForwardRef(Themed.Button)
    containerStyle={
      Object {
        "height": 64,
        "paddingVertical": 8,
      }
    }
    title="Agregar Artista"
  />
</LayoutView>
`;
